File: template/template.java
import java.util.Scanner;

public class template {

  public static void main(String[] args) {

    // TODO edit this code, this code is for
    // https://atcoder.jp/contests/practice/tasks/practice_1

    // param
    Scanner sc = new Scanner(System.in);
    int a = Integer.parseInt(sc.next());
    int b = Integer.parseInt(sc.next());
    int c = Integer.parseInt(sc.next());
    String s = sc.next();
    sc.close();

    // resolve
    int ans = a + b + c;

    // answer
    System.out.println(ans + " " + s);
  }

}

File: template/template.ts
import * as fs from "fs";

// util for input
// prettier-ignore
const lineit = (function* () { for (const line of fs.readFileSync(process.stdin.fd, "utf8").split("\n")) yield line.trim(); })();
// prettier-ignore
const wordit = (function* () { while (true) { let line = lineit.next(); if (line.done) break; for (const word of String(line.value).split(" ")) yield word; } })();
// prettier-ignore
const read = () => String((wordit.next()).value);

// main
const main = function () {
    // TODO edit this code, this code is for https://atcoder.jp/contests/practice/tasks/practice_1

    // param
    let a: number = Number(read());
    let b: number = Number(read());
    let c: number = Number(read());
    let s: string = read();

    // solve
    let ans = `${a + b + c} ${s}`;

    // answer
    console.log(ans);

    return;
};
main();
